holo-isis/src/packet/tlv.rs

```diff
@@ -1408,6 +1456,11 @@ impl Ipv6ReachTlv {
             buf.put(&entry.prefix.ip().octets()[0..plen_wire]);
 
             // Encode Sub-TLVs.
+
+            // Encode BIER Sub-TLVs.
+            entry.sub_tlvs.bier.iter().for_each(|entry| {
```

Comment:
nit: I'd prefer using a regular for loop to be consistent with the rest of the codebase.

Also, I think we can remove the "Encode BIER Sub-TLVs" comment. The "Encode Sub-TLVs" comment above it should suffice for all types of Sub-TLVs (e.g. BIER, SR-MPLS, SRv6, etc).